README.md

```diff
@@ -44,18 +44,17 @@ this will check that all figures are complete and indicate possible missing ones
 for slide view:
 
 ```
-./render_quarto.sh [BRIEFING_DATE] preview slides
+quarto render --profile slides
+
 ```
 
 for website view:
 
 ```
-./render_quarto.sh [BRIEFING_DATE] preview website
+quarto render --profile website
 ```
 
-### Build Everything
-
+You can also run quarto in preview mode, which will open a browser and navigate to the most recently modified file:
```

Comment:
The entire section was about preview, so this comment is a bit confusing...

Reply:
I think it was a combination of leaving the outdated subtitle und copy & pasting the last command... 😬 

My idea for the readme was to follow a _i)_ init with `wbcli`, _ii)_ render everything with quarto, _iii)_ if you feel like it use the preview structure. The updated version hopefully does that